CMakeLists.txt

```diff
@@ -0,0 +1,230 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(TheStdandardLib
+    VERSION 1.0
+    LANGUAGES C
+)
+
+option(TARGET_X86_64
+    "Default target device is set to x86_64"
+    "ON"
+)
+
+set(CMAKE_ANDROID_ARCH_ABI "unsupported_cpu_architecture")
```

Comment:
Is this a cmake-ism where it uses Android extensions to implement cross-platform builds?